cli/command/node/inspect_test.go

```diff
@@ -0,0 +1,120 @@
+package node
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/swarm"
+	"github.com/docker/docker/cli/internal/test"
+	// Import builder to get the builder function as package function
+	. "github.com/docker/docker/cli/internal/test/builder"
+	"github.com/docker/docker/pkg/testutil/assert"
+	"github.com/docker/docker/pkg/testutil/golden"
+)
+
+func TestNodeInspectShouldReturnAnErrorIfAPIfail(t *testing.T) {
```

Comment:
`Fail` should be uppercase.

Perhaps shorten the name to `TestNodeInspectErrorOnAPIFailure`